projects/plugins/protect/src/js/components/threats-list/paid-list.jsx

```diff
@@ -195,11 +209,8 @@ const PaidList = ( { list } ) => {
 						diff,
 						filename,
 						firstDetected, // todo: still needs a proper fix
```

Comment:
You mentioned previously that there might be casing conversion function in the redux store somewhere, if even necessary anymore we'll want to ensure we get this part fixed up with that or remove it if not.